portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase

```diff
@@ -642,18 +642,18 @@ definition {
 		var attributesDataTypeAscending1 = '''
 			male,agreedToTermsOfUse,defaultUser,emailAddressVerified,
 			birthday,createDate,modifiedDate,createDate,modifiedDate,status,classNameId,classPK,contactId,
-			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,companyId,contactId,facebookId,
-			ldapServerId,portraitId,timeZoneId,userId,emailAddress,employeeNumber,facebookSn,hoursOfOperation,
-			jabberSn,jobClass,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,firstName,
-			firstName,googleUserId,greeting,jobTitle,languageId,lastName,lastName,middleName,middleName,openId
+			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,contactId,facebookId,
```

Comment:
Hey @joshitagaki, I think that after `classPK` it should be `companyId` and then `contactId`. Could you also check and update the list?

![image](https://user-images.githubusercontent.com/76956227/223573760-1c57a947-e56a-409c-b223-9cb03756eb78.png)

Reply:
Thanks!